Source/WebCore/platform/graphics/filters/FilterOperations.cpp

```diff
@@ -110,6 +110,21 @@ IntOutsets FilterOperations::outsets() const
     return totalOutsets;
 }
 
+IntOutsets FilterOperations::accumulatedFilterOutsets(IntOutsets parentOutsets) const
+{
+    auto outsets = this->outsets();
+    if (outsets.isZero())
+        return parentOutsets;
+    if (parentOutsets.isZero())
+        return outsets;
+
+    auto top = std::max(outsets.top(), parentOutsets.top());
+    auto right = std::max(outsets.right(), parentOutsets.right());
+    auto left = std::max(outsets.left(), parentOutsets.left());
+    auto bottom = std::max(outsets.bottom(), parentOutsets.bottom());
+    return { top, right, bottom, left };
+}
```

Comment:
Is this the correct way to combine outsets for vertical or flipped blocks writing modes?